gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -121,15 +147,9 @@ def _is_device_sharing_supported(self):
         return self.supports_device_sharing and self.supports_hotplug
 
     def _create_management_interface(self, device_data, network_handler=None):
-        try:
-            token = (device_data['token']
-                     if device_data.get('token')
-                     else self.identity_handler.get_admin_token())
-        except Exception:
-            self._increment_stats_counter('keystone_token_get_failures')
-            LOG.error(_LE('Failed to get token for management interface'
-                          ' creation'))
-            return None
+        token = self._get_token(device_data.get('token'))
+        if not token:
+            return False
```

Comment:
Orchestration driver should return either None on failure/excetion, or non-None on success